githubpoxy.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"io"
+	"time"
+
+	"github.com/go-rod/rod"
+)
+
+type GitHubProvider struct {
+}

```

Comment:
Good to see you didn't go overboard, no need to start jamming properties in here yet